drivers/hmis_csv_twenty_twenty/spec/models/clean_up_move_in_dates_spec.rb

```diff
@@ -0,0 +1,53 @@
+require 'rails_helper'
```

Comment:
Can we run these with and without the cleanup enabled, just to confirm that the move-in-dates don't get stripped if the cleanup isn't enabled?